### DISCUSSION TITLE
[v4.0.18]标签显示错误

### DISCUSSION THREAD

**Discussion by Angel0726** [tangly1024/NotionNext#1992] [Q&A]
**描述bug**
【此项必填】简单说明bug的现象、相关的错误提示、日志等

 ![Screenshot_2023-11-15-19-27-33-409_com.android.browser.jpg](https://github.com/tangly1024/NotionNext/assets/32816780/1600f9b2-d665-45ef-8bd6-584ee329c756)

https://www.vabc.top/
**复现步骤**
【此项必填】出现这个bug的操作步骤

**期望的正常结果**
【此项必填】希望按这个步骤，正常操作结果是什么

**截图**
【可选】相关的页面，应该的结果

**环境**

- 【必填】NotionNext版本 [例如. 4.0.18]
- 【必填】主题 [例如. hexo]
- 【可选】操作系统: [例如. iOS, Android, macOS, windows]
- 【可选】浏览器 [例如. chrome, safari, firefox]

**补充说明**
【可选】与问题相关的其它说明

**Comment by tangly1024** [comment]
升级至最新版已修复此问题；另外，有一些notion的emoji在网页上是不支持的，请替换掉即可。
